client/src/components/Login/LoginPage.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, notification, Typography } from "antd";
+import axios from "axios";
+import { parse, stringify } from "querystring";
+import React from "react";
+import { Redirect } from "react-router-dom";
+
+import { loginPath, LoginResponse } from "../../api";
+
+const { Title } = Typography;
+
+type LoginPageProps = {
+  loggedIn: boolean;
+  setLoggedIn: (value: boolean) => void;
+};
+
+const LoginPage = ({ loggedIn, setLoggedIn }: LoginPageProps): JSX.Element => {
+  function openNotification(errorMessage: string) {
```

Comment:
Piyush likes explicitly defining when something is a method vs when something is a variable because of his smol java brain.
(No reason lol)
And moved outside - do you think it looks better before or after props and/or the exported function component?

Reply:
After function component might be interesting since we don't really read the prop type code as much. We can discuss later.